benchmark/createBenchmark.js

```diff
@@ -1,22 +1,29 @@
+// eslint-disable-next-line node/no-extraneous-require
```

Comment:
@ooflorent shall we import webpack relatively here same as we do in tests to eliminate this issue as well?

Reply:
Go ahead!